wdl2cwl/main.py

```diff
@@ -643,6 +656,26 @@ def convert(workflow: str) -> str:
             )
         )
 
+    if "disks" in ast.task_runtime:
+        # get the outdirMin from runtime of disks
+        outdir_min: Union[str, int] = ""
+
+        outdir_min = get_outdir_min(ast.task_runtime["disks"])
+
+        requirements.append(
+            cwl.ResourceRequirement(
+                outdirMin=outdir_min,
+            )
+        )
+    elif "disks" not in ast.task_runtime:
```

Comment:
```suggestion
    elif:
```
Redundant to check again here